src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -168,8 +199,29 @@ class ReferralService(
     return referralAccessFilter.serviceProviderReferralSummaries(referralSummaries, user)
   }
 
+  private fun getSentReferralSummaryForProbationPractitionerUser(user: AuthUser, sentReferralFilterSpecification: Specification<ReferralSummary>, page: Pageable?): Iterable<ReferralSummary> {
+    var referralsForPPUser = ReferralSpecifications.createdBy<ReferralSummary>(user)
+    try {
```

Comment:
it would also be good to do some de-duplication here if possible. not a blocker though.

Reply:
i know the plan is to remove the deprecated code in future, but sometimes these things have a habit of sticking around for longer than planned... :)